src/axes.jl

```diff
@@ -241,7 +241,7 @@ function get_ticks(axis::Axis)
     elseif typeof(ticks) <: Union{AVec, Int}
         # override ticks, but get the labels
         optimal_ticks_and_labels(axis, ticks)
-    elseif typeof(ticks) <: NTuple{2}
+    elseif typeof(ticks) <: NTuple{2, Any}
```

Comment:
I accidently commited the "passing tuple to ticks on 0.6" fix on this wip branch instead of a new one yesterday. Then I undid it again. Will this cause conflicts?

Also, I think we have more important stuff to deal with right now than this new feature and I think I will not work on this in the near future. Is there anything speaking against me closing this wip PR and opening a new one when I find time to continue working on it, in order to avoid these kind of "accidents"?